l2l/utils/JUBE_runner.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, trajectory):
         self.trajectory = trajectory
         self.done = False
         if 'JUBE_params' not in self.trajectory.par.keys():
-            print("exception")
+            raise Exception("The trajectory must contain JUBE_params")
```

Comment:
`raise KeyError` suits here better than a generic error.

Reply:
Ok, I will change this accordingly